webpack.config.js

```diff
@@ -3,49 +3,99 @@
 'use strict';
 
 var path = require('path'),
-    Webpack = require('webpack'),
-    outputRoot = process.env.OUTPUT_ROOT !== undefined ? process.env.OUTPUT_ROOT : 'pldoc/public/',
-    siteRoot = process.env.SITE_ROOT !== undefined ? process.env.SITE_ROOT : '/',
-    publicJavaScriptRoot = 'public/';
+    webpack = require('webpack'),
+    ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
Added a bunch more documentation to this file about what all the different settings and paths are doing, let me know what you think.